aries_vcx/src/handlers/proof_presentation/prover.rs

```diff
@@ -134,6 +134,7 @@ impl Prover {
             .presentation_request()?
             .request_presentations_attach
             .content()
+            .map_err(|err| err.into())
```

Comment:
seems superflous, converting VcxError to VcxError

Reply:
ah I see ,that relates to my first comment about reworking Error in messages crate. Can we at least rename this "messages" VcxError/VcxResult to something like `MessagesError/MessagesResult`? FYI Intelij can do this mass rename on 2 clicks.